contrib/opencensus-ext-azure/CHANGELOG.md

```diff
@@ -2,6 +2,14 @@
 
 ## Unreleased
 
+## 1.0.0
+Released 2019-09-30
+
+- Standard Metrics - Incoming Requests Execution Time
+  ([#773](https://github.com/census-instrumentation/opencensus-python/pull/773))
+- Implement connection strings
+  ([#767](https://github.com/census-instrumentation/opencensus-python/pull/767))
```

Comment:
Ideally these would already be in the changelog on master at the time of the release.

Reply:
Good point. I will update the changelog on master ASAP.

Reply:
[#794]